include/lsst/sphgeom/Pixelization.h

```diff
@@ -23,6 +23,11 @@
 #ifndef LSST_SPHGEOM_PIXELIZATION_H_
 #define LSST_SPHGEOM_PIXELIZATION_H_
 
+// PyInt_GetMax is not in the python 3 C API, so we define it for python >=3
+#if PY_MAJOR_VERSION >= 3
+#define PyInt_GetMax(x) 2<<31
+#endif
```

Comment:
@smonkewitz do you have an opinion on this? It seems a bit weird to use 32bits when even on Python 2 we have 64bit ints in 64bit mode.